config/action.d/nftables-common.conf

```diff
@@ -0,0 +1,96 @@
+# Fail2Ban configuration file
+#
+# Author: Daniel Black
+# Author: Cyril Jaquier
+# Modified: Yaroslav O. Halchenko <debian@onerussian.com>
+# 			made active on all ports from original iptables.conf
+# Modified: Alexander Belykh <albel727@ngs.ru>
+#                       adapted for nftables
+#
+# This is a included configuration file and includes the definitions for the nftables
+# used in all nftables based actions by default.
+#
+# The user can override the defaults in nftables-common.local
+
+[INCLUDES]
+
+after = nftables-common.local
+
+[Definition]
+
+# Option:  nftables_mode
+# Notes.:  additional expressions for nftables filter rule
+# Values:  nftables expressions
+#
+nftables_mode = <protocol> dport { <port> }
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart = <nftables> add set filter f2b-<name> { type ipv4_addr\; }
```

Comment:
actually seems to work in bash and dash but not zsh (with escapes works everywhere)... and by "works" I mean just that it doesn't fail in syntax, since probably I do not really use nftables yet and still iptables, I just get:
```
# nft add set filter f2b-test \{ type ipv4_addr\; \}
<cmdline>:1:1-43: Error: Could not process rule: Table 'filter' does not exist
add set filter f2b-test { type ipv4_addr; }
```
is that a table we might need to setup or a normal system should have it?